Chorelord/ChoreListController.swift

```diff
@@ -8,21 +8,60 @@
 
```

Comment:
We typically keep "View" in the name, so `ChoreListViewController` would be a more consistent name with other VCs in other projects.  If you update it, be sure to update both the class name and the filename.